src/agenteval/models.py

```diff
@@ -31,59 +42,50 @@ class SubmissionMetadata(BaseModel):
     openness: str | None = None
     tool_usage: str | None = None
 
-
-class EvalResult(EvalConfig):
+class TaskResults(BaseModel):
```

Comment:
This class just holds the list of `TaskResult`s, with some utility methods used for validation